docs/intentReaderHandlers/RegisterEventHandler.md

```diff
@@ -0,0 +1,342 @@
+# RegisterEventHandler 
+
+## Overview
+
+RegisterEventHandler is invoked when the task specified in the intent has the value "registerEvent". This handler is used to register for the event in FCA.
+
+It performs the following actions :
+1. Parse the input message received to get the event.
+2. Register the event passed in params. 
+3. Save the response/error and perform schema validations. More about schema validations here : [validation documentation](../Validations.md).
+4. Format the result and send the response back to the IntentReader.
```

Comment:
Same comment as others 
Parses
Registers the event passed in the params.
Saves
Formats the result and sends

Reply:
updated for all handlers